src/math/mat4.js

```diff
@@ -4,6 +4,11 @@ import { Vec3 } from './vec3.js';
 import { Vec4 } from './vec4.js';
 
 var _halfSize = new Vec2();
+var x = new Vec3();
```

Comment:
just curious if these could be inside class now as static or similar?

Reply:
also, will these variables with names "x" "y" "z" create possible conflicts in the rest of the class?

Reply:
We could do. I think I'd probably put them on the `Vec3` class as `Vec3._temp1`, `Vec3._temp2`, `Vec3._temp3` maybe. But I personally don't have a big problem with these global to the module. It's kind of nice because the app code cannot access them (and therefore misuse them), which isn't true of they're on the class.

Reply:
And no, conflicts aren't possible. Rollup renames variables that conflicts when building the bundled library. It's pretty cool!

Reply:
I mean more like conflict with a local variable inside function of Vec3. Few functions have "var x". If I don't have this var line would x reference the global x?

Reply:
Yes.